userbenchmark/group_bench/configs/torch_ao.yaml

```diff
@@ -0,0 +1,13 @@
+model: "*"
+test: eval
+device: cuda
+extra_args: --precision bf16 --torchdynamo inductor --inductor-compile-mode max-autotune
```

Comment:
@HDCharles In this config, we define the baseline extra_args being `--precision bf16 --torchdynamo inductor --inductor-compile-mode max-autotune`, so it will apply this to every test_group/subgroup defined below.

Reply:
got it, is it possible to see the result of a run so i can see what is catalogued?